2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_setup.py:_flush():75] Loading settings from /home/mgatlin3/.config/wandb/settings
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'sweep-FT', 'entity': 'mgatlin3', 'root_dir': '/home/mgatlin3/FT', 'sweep_id': 'mny6c5uf', 'run_id': '8egrhygs', 'sweep_param_path': '/home/mgatlin3/FT/wandb/sweep-mny6c5uf/config-8egrhygs.yaml'}
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sweep.py', 'program': '/home/mgatlin3/FT/sweep.py'}
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_init.py:_log_setup():437] Logging user logs to /home/mgatlin3/FT/wandb/run-20220818_163630-8egrhygs/logs/debug.log
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_init.py:_log_setup():438] Logging internal logs to /home/mgatlin3/FT/wandb/run-20220818_163630-8egrhygs/logs/debug-internal.log
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_init.py:init():471] calling init triggers
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_init.py:init():474] wandb.init called with sweep_config: {'batch_size': 4, 'dropout_c': 0.1, 'dropout_ga': 0, 'epochs': 40, 'learning_rate': 0.001, 'num_features_c1': 32, 'num_features_c2': 128, 'num_features_c3': 32, 'num_features_c4': 128}
config: {'batch_size': 4, 'epochs': 5, 'dropout_c': 0.3, 'dropout_ga': 0.1, 'num_features_c1': 64, 'num_features_c2': 64, 'num_features_c3': 128, 'num_features_c4': 64, 'learning_rate': 0.0003}
2022-08-18 16:36:30,917 INFO    Thread-2  :252135 [wandb_init.py:init():524] starting backend
2022-08-18 16:36:30,918 INFO    Thread-2  :252135 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-18 16:36:30,933 INFO    Thread-2  :252135 [backend.py:ensure_launched():217] starting backend process...
2022-08-18 16:36:30,940 INFO    Thread-2  :252135 [backend.py:ensure_launched():222] started backend process with pid: 252161
2022-08-18 16:36:30,941 INFO    Thread-2  :252135 [wandb_init.py:init():533] backend started and connected
2022-08-18 16:36:30,948 INFO    Thread-2  :252135 [wandb_run.py:_config_callback():1131] config_cb None None {'batch_size': 4, 'dropout_c': 0.1, 'dropout_ga': 0, 'epochs': 40, 'learning_rate': 0.001, 'num_features_c1': 32, 'num_features_c2': 128, 'num_features_c3': 32, 'num_features_c4': 128}
2022-08-18 16:36:30,951 INFO    Thread-2  :252135 [wandb_init.py:init():597] updated telemetry
2022-08-18 16:36:30,971 INFO    Thread-2  :252135 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-08-18 16:36:31,779 INFO    Thread-2  :252135 [wandb_run.py:_on_init():1923] communicating current version
2022-08-18 16:36:31,871 INFO    Thread-2  :252135 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-18 16:36:31,871 INFO    Thread-2  :252135 [wandb_init.py:init():659] starting run threads in backend
2022-08-18 16:36:36,879 INFO    Thread-2  :252135 [wandb_run.py:_console_start():1897] atexit reg
2022-08-18 16:36:36,880 INFO    Thread-2  :252135 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-08-18 16:36:36,880 INFO    Thread-2  :252135 [wandb_run.py:_redirect():1775] Redirecting console.
2022-08-18 16:36:36,881 ERROR   Thread-2  :252135 [wandb_run.py:_redirect():1834] Failed to redirect.
Traceback (most recent call last):
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1827, in _redirect
    out_redir.install()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/mgatlin3/.local/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/mgatlin3/.conda/envs/poop/lib/python3.9/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-08-18 16:36:36,884 INFO    Thread-2  :252135 [wandb_init.py:init():684] run started, returning control to user process
2022-08-18 16:36:36,884 INFO    Thread-2  :252135 [wandb_config.py:__setitem__():150] config set architecture_name = CNN - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f655c66f730>>
2022-08-18 16:36:36,884 INFO    Thread-2  :252135 [wandb_run.py:_config_callback():1131] config_cb architecture_name CNN None
2022-08-18 17:14:47,505 INFO    Thread-2  :252135 [wandb_run.py:_finish():1685] finishing run mgatlin3/sweep-FT/8egrhygs
2022-08-18 17:14:47,506 INFO    Thread-2  :252135 [wandb_run.py:_atexit_cleanup():1866] got exitcode: 0
2022-08-18 17:14:47,509 INFO    Thread-2  :252135 [wandb_run.py:_restore():1838] restore
2022-08-18 17:14:47,512 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1410098
  total_bytes: 1410098
}

2022-08-18 17:14:47,617 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1410098
  total_bytes: 1410098
}

2022-08-18 17:14:48,220 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1410098
  total_bytes: 1410098
}

2022-08-18 17:14:48,321 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 3378
  total_bytes: 1419928
}

2022-08-18 17:14:48,423 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 3378
  total_bytes: 1419928
}

2022-08-18 17:14:48,524 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 62360
  total_bytes: 1419928
}

2022-08-18 17:14:48,626 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 62360
  total_bytes: 1419928
}

2022-08-18 17:14:48,727 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 283544
  total_bytes: 1419928
}

2022-08-18 17:14:48,829 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:48,930 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,032 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,134 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,236 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,338 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,439 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,619 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}

2022-08-18 17:14:49,830 INFO    Thread-2  :252135 [wandb_run.py:_on_finish():1995] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 1419928
  total_bytes: 1419928
}
local_info {
}

2022-08-18 17:14:51,275 INFO    Thread-2  :252135 [wandb_run.py:_footer_history_summary_info():3102] rendering history
2022-08-18 17:14:51,276 INFO    Thread-2  :252135 [wandb_run.py:_footer_history_summary_info():3134] rendering summary
2022-08-18 17:14:51,278 INFO    Thread-2  :252135 [wandb_run.py:_footer_sync_info():3057] logging synced files
